src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,12 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    use_bluestore=false
```

Comment:
@tchaikov 
Good pointer, thanx.

Will update the PR and commit with this info.
And I'll change the nameing.